audio/audio_driver.c

```diff
@@ -385,12 +390,16 @@ bool audio_driver_find_driver(
 }
 
 /**
```

Comment:
An earlier draft had mic support as part of the audio driver; since I (thought I) needed to edit this function, I needed to understand it. So I made a bunch of comments as I went.